apps/explorer/src/ui/stories/SenderRecipient.stories.tsx

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { type Meta, type StoryObj } from '@storybook/react';
+import { MemoryRouter } from 'react-router-dom';
+
+import { SenderRecipient, type SenderRecipientProps } from '../SenderRecipient';
+
+const data = {
```

Comment:
This looks like it's only used in one example, and only the `sender` is actually used (all other are overwritten). You can just inline that below.